src/main/app/src/app/informatie-objecten/informatie-object-view/informatie-object-view.component.html

```diff
@@ -31,7 +31,7 @@
                                     <zac-static-text [label]="'indicatieGebruiksrecht' | translate"
                                                      [value]="infoObject.indicatieGebruiksrecht ? 'Ja' : 'Nee'"></zac-static-text>
                                     <zac-static-text [label]="'taal' | translate"
-                                                     [value]="'taal.'+infoObject.taal | translate"></zac-static-text>
+                                                     [value]="infoObject.taal"></zac-static-text>
```

Comment:
je hebt de taal vertaling weggehaald dus nu zie je denk ik iets van nld, eng fry in plaats van Nederlands, Engels en Frysk. is dat gewenst?

Reply:
Ik Had de vertaling bewust weggehaald. Mij leek het geen goed idee om de omzetting van een ISO code naar een andere weergave vast te leggen in een message property file, Maar ik zet het toch maar evenb terug.